selfdrive/updated.py

```diff
@@ -437,9 +439,13 @@ def main() -> None:
       init_overlay()
 
       # ensure we have some params written soon after startup
-      updater.set_params(update_failed_count, exception)
+      updater.set_params(False, update_failed_count, exception)
       update_failed_count += 1
```

Comment:
shouldn't be incremented if time invalid

Reply:
done